bullet_train/app/models/concerns/teams/base.rb

```diff
@@ -27,10 +27,6 @@ module Teams::Base
       if defined?(Billing::Stripe::Subscription)
         has_many :billing_stripe_subscriptions, class_name: "Billing::Stripe::Subscription", dependent: :destroy, foreign_key: :team_id
       end
-
-      if defined?(Billing::Usage::TeamSupport)
-        include Billing::Usage::TeamSupport
-      end
```

Comment:
Alternatively, if we want to fire these lines we can update to this:

```ruby
     if defined?(Billing::Usage::HasTrackers)
        include Billing::Usage::HasTrackers
      end
```

However, we should do that in the billing usage gem after https://github.com/bullet-train-co/bullet_train-core/pull/335 ships.